docs/source/querying/querybuilder/append.rst

```diff
@@ -707,8 +707,7 @@ Let's take the above example, but put a filter on the label of the link and proj
     qb = QueryBuilder()
```

Comment:
In the text section above, it says something about the keyword being preceeded by '*link*', but this should be `edge`, right?

Reply:
Sorry, I did not find anything that is related to links in the text above. The only sentence that says about links below is:
> Another feature that had to be added are projections, filters and labels on the edges of the graphs, that is to say links or paths between nodes. It works the same way, just that the keyword is preceeded by ‘link’. Let’s take the above example, but put a filter on the label of the link and project the link label:

Reply:
sorry, you were right.